client/src/store/actions/adminLoginActions.js

```diff
@@ -0,0 +1,3 @@
+const SET_LOGIN = 'SET_LOGIN';
```

Comment:
I have seen this kind of `actions` files in redux examples. However, it seems a bit `overkill` to create a variable, assign it to a string and export it just so that you don't use a string when dispatching a type of action.

Reply:
Refer to Michaels tech demo ;D